thornodes/leaving.md

```diff
@@ -50,29 +58,84 @@ This ensures you can safely leave this system if you no longer have access to yo
+
+Example: If your node has a $5m bond \(in RUNE\), but has $1m in assets in its vaults it can't return, it will lose $1.5m in RUNE from its bond. The Node will only get back $3.5m of its bond. 
+{% endhint %}
+
+## Contingency: Manual Leave Procedure
+
+If your node is completely offline or destroyed, you will have to perform a manual return of Yggdrasil funds in order to prevent 1.5x bond fine. Ensure you have reviewed this procedure and have all tools ready to go in case you need to do it in *anger*.  This is a time-critical event - you have a few hours to return all funds before the network assumes you have stolen them.  
 
-Example: If your node has a $500k bond \(in RUNE\), but has $100k in assets in its vaults it can't return, it will lose $150k in RUNE from its bond. The Node will get back $350k in its bond. 
+{% hint style="info" %}
+If you do not perform all of these steps in time, your bond will be fined 1.5x stolen funds. The remaining funds belonging to Yggdrasil `make mnemonic` are now yours; but you just paid a 50% premium for them, losing a lot of money. You can use the following procedure in a similar way to recover these funds. 
 {% endhint %}
 
+**Requirement**: You have your `make mnemonic` Yggdrasil mnemonic available. If you do not have this, you cannot manually return funds. 
+
+Options: 
+1. *Coming Soon*: Use ASGARDEX for Manual Return.   
+2. *Coming Soon*: Check Discord Dev channels for manual return cli tool.  
+3. Extract Private Key + Manual return each asset using wallets:  
+
+Your Yggdrasil `make mnemonic` phrase is used to generate the `m/44'/931'/0'/0/0` private key which is used for *all* chains. Pasting the mnemonic into common wallets will not work as they will be looking under a different "standard" HD Path. Instead, go to [https://iancoleman.io/bip39/](https://iancoleman.io/bip39/) and paste in your mnemonic, select *RUNE* from the Dropdown list and in the bottom table, copy the `m/44'/931'/0'/0/0` private key string. Use this to import into wallets.  
+
+The next step is to find the latest inbound addresses. Use [https://thornode.thorchain.info/thorchain/inbound_addresses](https://thornode.thorchain.info/thorchain/inbound_addresses)
+
+{% hint style="warning" %}
+**Do not cache** inbound_addresses. These are only valid for a short period of time. Always refresh to get the latest before sending funds.  
+{% endhint %}
+
+{% hint style="warning" %}
+Use the **address** field. Chains with a **router** present such as ETH need to send funds via the router smart-contract. Paste the router address into etherscan, click "Contract" and "Write Contract" and use a Web3 wallet to connect. 
+{% endhint %}
+
+The **memo** required is `YGGDRASIL-:<BlockHeight>`. For example `YGGDRASIL-:782412`. The block height can be found from the `status_since` field here: 
```

Comment:
If I'm not wrong, `memo` has to be hex encoded (at least by using MetaMask - [web3.toHex](web3.toHex) helps here, you can encode the memo by using `web3.toHex` online https://www.mobilefish.com/download/ethereum/web3api.html)